src/lib/stores/banner.ts

```diff
@@ -1,7 +1,12 @@
+import { get } from 'svelte/store';
 import { persistStore } from '$lib/stores/persist-store';
 
-export const closedBannerId = persistStore('closedBannerId', null);
+export const closedBanners = persistStore('closedBanners', JSON.stringify([]));
```

Comment:
Maybe `persistStore` should just handle stringifying and parsing since it's _always_ writing to `localStorage`?

Reply:
yea sg, making persistStore generic

Reply:
yea sg, made `persistStore` generic and handle serialization